src/drivers/pmw3901/pmw3901.cpp

```diff
@@ -844,17 +892,16 @@ int
 pmw3901_main(int argc, char *argv[])
 {
 	int myoptind = 1;
-	uint8_t rotation = distance_sensor_s::ROTATION_DOWNWARD_FACING;
 
-	if (argc < 2) {
+	if (argc < 1) {
```

Comment:
@RomanBapst for this driver we don't have the px4 argument check and I think that with arg = 1 we would get a segfault because argv[myoptind] is null. I will double check this tomorrow. 
What I mean is if I just write pmw3901 in the console with no command.

Reply:
@DanielePettenuzzo Ah ok, good catch.

Reply:
Done